chain-signatures/node/src/indexer.rs

```diff
@@ -260,9 +264,6 @@ async fn handle_block(
     }
     drop(queue);
 
-    if block.block_height() % 1000 == 0 {
-        tracing::info!(block_height = block.block_height(), "indexed block");
-    }
```

Comment:
we should keep this as it was nice to see if indexer was still running or not. It doesn't spam the logs either

Reply:
ok to keep this one. It's helpful for us to see what are the logs around the time block height increase stopped.

Reply:
I'm not fond of such a pattern, since it is inconsistent. What if 1 block dropped? What if we need to know about every block we are receiving? I've changed the message to eliminate those.

And I've added a debug!() log that is printing all the new blocks.